client/state/sites/selectors.js

```diff
@@ -1020,3 +1020,15 @@ export const hasDefaultSiteTitle = ( state, siteId ) => {
 	// we are using startsWith here, as getSiteSlug returns "slug.wordpress.com"
 	return site.name === i18n.translate( 'Site Title' ) || startsWith( slug, site.name );
 };
+
+/**
+ * Returns true if the site supports managing Jetpack settings remotely.
+ * False otherwise.
+ *
+ * @param {Object} state  Global state tree
+ * @param {Object} siteId Site ID
+ * @return {?Boolean}     Whether site supports managing Jetpack settings remotely.
+ */
+export const siteSupportsCalypsoSettingsUI = ( state, siteId ) => {
+	return isJetpackMinimumVersion( state, siteId, '4.5.0' );
```

Comment:
My intention here is to keep this selector intentionally nullable, just so it is consistent with `isJetpackMinimumVersion`, but also to be able to differentiate between "not supported" and "not known".